src/App.jsx

```diff
@@ -1,13 +1,33 @@
+import React, { useState, useEffect } from "react";
 import "./styles/app.scss";
 import Header from "./components/Header";
 import Banner from "./components/Banner";
 import Modal from "./components/Modal";
 import RoomSelect from "./components/RoomSelect";
 import Range from "./components/Range";
 import images from "./helpers/roomImages";
+import Catalogue from "./components/Catalogue";
+import Product from "./components/Product";
```

Comment:
```suggestion
import React, { useState, useEffect } from "react";

import Header from "./components/Header";
import Banner from "./components/Banner";
import Modal from "./components/Modal";
import RoomSelect from "./components/RoomSelect";
import Range from "./components/Range";
import Catalogue from "./components/Catalogue";
import Product from "./components/Product";

import images from "./helpers/roomImages";

import "./styles/app.scss";
```

Just to give you an idea, i personally like to group the imports by the type of it (Library, components, styles...)

Reply:
That's true; it would look much cleaner.